how-to-organize-a-meetup.md

```diff
@@ -0,0 +1,133 @@
+# How to plan a meetup
+
+This document is meant to help you plan a meetup. It is specifically tailed for the IPFS community; however, in order to make it more general, I have put every IPFS-only task into its own section.
+
+## Pre-planning
+
+Before hosting a meetup, you should identify: 
+
+- Who is the meetup for? Beginners? Advanced users? Advanced Developers?
+- What topics should be covered?
+- What are your core values, and how do you want these to come off? Inclusiveness;
+- What is the tone? Will this be a hangout group, or a group for intense discussions, or a tutorial, or a hack group?
+- What will the format be?
+- How often do you plan on running it?
+- Who is able to run it with you?
+- What will the name of the meetup be?
+- How hyperlocal will this be? What is your intended geographical spread?
+
+You should:
+
+- Attend other meetups
+- Meet in person to discuss the above list, and write down your answers for them
+
+### Defining Roles
+
+Splitting up organizational tasks into a few different, well-defined roles will help with making sure that everyone is able to do their best, and that there are clear expectations and exits for those who can't help out.
+
+Have one person who is responsible and in charge of each of the following:
+
+- Facilitator
+- Master of Ceremonies
+- Venue
+- Sponsors
+- Food and drink
+- Social media - Before and During
+- Website // Meetup Group // Mailing List
+- Post-event blogger
+
+### Prerequisites
+
+Meetups take a lot of effort. Here are some things you should set up before a meetup is planned:
+
+- A Meetup group. These can cost some initial money. Another option is Eventbrite.
+- A Twitter account.
+- A Newsletter. Tinylette or Mailchimp are good for this.
+- A way for speakers to submit talks.
+- A way for the facilitators and committee to approve talks.
+- A ticketing system, if needed (Meetup Works for this.)
+
+### Date and Location
+
+Once you have the date and location generally specced out, there are a few things to do: 
+
+- Make sure that this isn't overlapping with any other similar events, at the same time or in the same location, and that it isn't too close to other large events.
+- Add the event to a MeetUp group. If there isn't a MeetUp group, create one.
+- Post about the event on Twitter
+  - Plan out tweets for the event on a regular basis.
+- Send out a message to the newsletter.
+
+#### IPFS
```

Comment:
I was trying to make a more general guide. See line 3:

> This document is meant to help you plan a meetup. It is specifically tailed for the IPFS community; however, in order to make it more general, I have put every IPFS-only task into its own section.

This wasn't clear enough; should I just mix them in, and make a separate document? It might be smart.